selfdrive/test/test_routes.py

```diff
@@ -251,20 +251,11 @@
     'enableCamera': True,
     'enableDsu': False,
   },
-  "b0f5a01cf604185c|2018-01-26--00-54-32": {
+  "4e45c89c38e8ec4d|2021-05-02--02-49-28--15": {
     'carFingerprint': TOYOTA.COROLLA,
     'enableCamera': True,
     'enableDsu': True,
-  },
-  "b0f5a01cf604185c|2018-01-26--10-54-38": {
-    'carFingerprint': TOYOTA.COROLLA,
-    'enableCamera': True,
-    'enableDsu': False,
-  },
-  "b0f5a01cf604185c|2018-01-26--10-59-31": {
-    'carFingerprint': TOYOTA.COROLLA,
-    'enableCamera': False,
-    'enableDsu': False,
+    'enableGasInterceptor': True,
```

Comment:
no need for this, it's not used anymore

Reply:
Should enableDsu be false since I've unplugged mine?

Reply:
you can just leave out everything besides carFingerprint